### DISCUSSION TITLE
Is it possible to hide Usage by plan?

### DISCUSSION THREAD

**Discussion by fu-sen** [vercel/vercel#6068] [Ideas] [answered]
I use ▲Vercel as a hobby, so the Free Plan is sufficient. 😁

Due to a recent change, if you select **Usage** in your Free Plan, you will get this:
![スクリーンショット 2021-04-05 10 30 09](https://user-images.githubusercontent.com/5434159/113528000-452ef500-95fa-11eb-9efb-0402d3720f76.jpg)

This is because ▲ vercel is prepared for paid plans, so you don't need to see it in Free Plan.
But then can ▲Vercel members hide this from the menu? It can be wasteful and unpleasant.

**Comment by styfle** [comment, marked as answer]
Hi @fu-sen 

The Usage page is available for Hobby users now, thanks!

https://vercel.com/changelog/detailed-usage-metrics-for-personal-accounts

**Reply by fu-sen** [reply]
In particular, by knowing the Bandwidth of the app you are running on Vercel,
you can prevent notifications due to restrictions from Vercel.
We users can use Vercel with more peace of mind. Great!
